### ISSUE TITLE
benchmarks_ensebles_scores` is missing a column

### ISSUE THREAD

**Issue by robkooper** [PecanProject/bety#435]
from @bcow : I just found a problem in bety that I could have sworn we already fixed ` benchmarks_ensebles_scores` is missing a column: it needs `t.text :score, :null => false`

**Comment by robkooper** [comment]
Following are the migrations (in sql)
```
ALTER TABLE "benchmarks_ensembles_scores" ADD COLUMN "score" text NOT NULL;
INSERT INTO "schema_migrations" ("version") VALUES ('20161005181021');
```

**Comment by gsrohde** [comment]
Fixed by release 4.13.
